go.mod

```diff
@@ -31,6 +31,10 @@ require (
 	sigs.k8s.io/kubefed v0.1.0-rc2
 )
 
+replace github.com/codeready-toolchain/api => github.com/MatousJobanek/api v0.0.0-20191008135136-1338bbea618f
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/MatousJobanek/toolchain-common v0.0.0-20191008140617-66eaba2d7197
```

Comment:
It's a just reminder (rather for myself :) that it should be removed before merging.

Reply:
done c50921c